app/Controller/ServersController.php

```diff
@@ -2527,4 +2527,7 @@ public function getAvailableSyncFilteringRules()
         $syncFilteringRules = $this->Server->getAvailableSyncFilteringRules($this->Auth->user());
         return $this->RestResponse->viewData($syncFilteringRules);
     }
+
+    public function openapi(): void {
```

Comment:
Perhaps remove the return promise for now - we'll definitely make use of those once we move to cake 4 but until then we still know of some happy PHP 5.6 users (cough) ;)